### ISSUE TITLE
ModuleNotFoundError: No module named 'sports'

### ISSUE THREAD

**Issue by dsaha21** [roboflow/sports#7]
Hi @SkalskiP, 

I was running the main.py inside the examples/soccer in my local system and getting an error like the following: 

<img width="338" alt="Roboflow-sports1" src="https://github.com/user-attachments/assets/e8c88685-d1e0-4e2b-a34b-852618a639bc">

I moved the main.py to outside along with setup.py
Fortunately it worked. I was thinking if possible you could give it a look.

Thanks

**Comment by SkalskiP** [comment]
Hi @dsaha21 did you follow installation steps?

**Comment by dsaha21** [comment]
Hello @SkalskiP,

I tried many times but unable to install the setup.py. Its showing exit code 1. 

<img width="663" alt="Roboflow-sports2" src="https://github.com/user-attachments/assets/e4773642-7baf-498f-827a-16bf7cf9662a">

(Py39) D:\>python --version
Python 3.9.16

(Py39) D:\>pip --version
pip 23.0.1 

Might be a version/ system issue. Sorry I cannot figure out. 
Actually I have few plans  to add some new features on the ball and the player tracking views. Thus it would be great if you look into it once 

Thanks again

**Comment by SkalskiP** [comment]
@dsaha21 looks like `pip install setuptools` should solve the problem.

**Comment by dsaha21** [comment]
@SkalskiP Getting the same above error. 

If possible, can be please mention the python, pip and setuptools version of your system, I can then install / upgrade the packages according to it. I think it might be a version mismatch, but not sure

**Comment by SkalskiP** [comment]
Python: 3.11.6
Pip: 23.2.1
Setuptools: 68.2.2

**Comment by SkalskiP** [comment]
Did it help?

**Comment by dsaha21** [comment]
Hi @SkalskiP, 

I was able to find out the error. I am using Windows OS. For the setup.py, 
1. I installed python 3.12.4
2. Made a virtual environment
3. If still showing error, separately add *pip install setuptools*
-------------
After this I was again facing the following error : 

> Getting requirements to build wheel did not run successfully.
> │ exit code: 1
> 
> return codecs.charmap_decode(input,self.errors,decoding_table)[0]
> ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> UnicodeDecodeError: 'charmap' codec can't decode byte 0x8f in position 3227: character maps to

**IT WAS DUE TO THE LINE  --> README = (HERE / "README.md").read_text() in the setup.py**

I commented the line and finally everything is installed along with all the model weights and videos. The videos are running along with the different modes -> pitch, player, ball detection.

I hope this will help for any people running on Windows OS, if any of them faces this error can view from here. 👍
